eventing/triggers/new-trigger-filters.adoc

```diff
@@ -0,0 +1,45 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="new-trigger-filters"]
+= New trigger filters
+include::_attributes/common-attributes.adoc[]
+:context: new-trigger-filters
+
+toc::[]
+
+:FeatureName: New trigger filters feature
+include::snippets/technology-preview.adoc[]
+
+The new trigger filters feature adds a new `filters` field to triggers that aligns with the filters API field defined in the CloudEvents Subscriptions API. It allows you to specify filter expressions, where each expression evaluates to `true` or `false` for each event. 
+
+The following example shows a trigger using the new filters field:
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  filters:
+    - cesql: "source LIKE '%commerce%' AND type IN ('order.created', 'order.updated', 'order.canceled')"
```

Comment:
I think on the initial introduction, I'd also not use the `cesql` - htis is a little complex. and the description does not directly talk about the query and provides no info on it

Reply:
Do you want me to remove `cesql` part?

Reply:
I referred to this example: https://knative.dev/v1.14-docs/eventing/experimental-features/new-trigger-filters/. It was present in the overview section.